### DISCUSSION TITLE
希望增加全屏时，输入弹幕的功能

### DISCUSSION THREAD

**Discussion by lyjhyjyes** [the1812/Bilibili-Evolved#1784] [功能建议 / Ideas]
目前全屏下没有输入弹幕的入口

**Comment by Coulomb-G** [comment]
能截图看看吗?不开脚本b站都可以在全屏模式下发弹幕

**Comment by lyjhyjyes** [comment]
https://imgtu.com/i/gCPyr9
截图如上，谷歌邮件里回复，试下能不能看见

Coulomb-G ***@***.***> 于2021年4月27日周二 下午5:14写道：

**Reply by lyjhyjyes** [reply]
> https://imgtu.com/i/gCPyr9 截图如上，谷歌邮件里回复，试下能不能看见 Coulomb-G ***@***.***> 于2021年4月27日周二 下午5:14写道：
> […](#)
> 能截图看看吗?不开脚本b站都可以在全屏模式下发弹幕 — You are receiving this because you authored the thread. Reply to this email directly, view it on GitHub <[#1784 (comment)](https://github.com/the1812/Bilibili-Evolved/discussions/1784#discussioncomment-663527)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/ANUEF7R55QQKBIO2EUUZGF3TKZ553ANCNFSM43USGMFQ> .

https://z3.ax1x.com/2021/04/27/gCPyr9.png 不是很会用图床

**Reply by Coulomb-G** [reply]
你开着其他的什么脚本?关了再看看?你的分辨率是多少?

**Reply by the1812** [reply]
浏览器是不是开缩放了

**Reply by lyjhyjyes** [reply]
> 浏览器是不是开缩放了

对的，原来开了缩放影响全屏弹幕输入

**Reply by lyjhyjyes** [reply]
> 你开着其他的什么脚本?关了再看看?你的分辨率是多少?

和脚本无关。原来不支持缩放屏幕，导致放大175%，无法显示全屏弹幕入口
